packages/crypto/src/tokenUtils.test.ts

```
@@ -0,0 +1,96 @@
+    let mockPrisma: any;
+    const testOrgId = 1;
+
+    beforeEach(() => {
+        mockPrisma = {
+            secret: {
+                findUnique: vi.fn(),
+            },
+        };
+
+        vi.clearAllMocks();
+        delete process.env.TEST_TOKEN;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)